### PR TITLE
Pack: Add unit tests

### PR DESCRIPTION
**Pull request by 1chandu**
For #

#### Background
- Verifying precession of pack module conversion of float to RGBA U8 to float.
- Add unit tests using Transform.

#### Change List
-  Pack: Add unit tests

### CODE REVIEW

File: test/modules/core/shadertools/pack.spec.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+const EPSILON = 1e-5;
+
+// failure when 1e-6
+// expected: 100.54000091552734 actual: 100.54000854492188 diff: 0.00000762939453125
+
+// failure when 1e-7
+// expected: 1.2640000581741333 actual: 1.2640001773834229 diff: 1.1920928955078125e-7
+
+registerShaderModules([pack]);
```
Comment (ibgreen) [inline test/modules/core/shadertools/pack.spec.js]: Shader modules are designed to be used without registration. They can be supplied directly as part of `modules` array.

File: test/modules/core/shadertools/pack.spec.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  const transform = new Transform(gl, {
+    sourceBuffers: {
+      inFloat: input
+    },
+    feedbackBuffers: {
+      outFloat: output
+    },
+    vs: VS,
+    varyings: ['outFloat'],
+    modules: ['pack'],
```
Comment (ibgreen) [inline test/modules/core/shadertools/pack.spec.js]: Replace with: `modules: [pack]`

File: test/modules/core/shadertools/pack.spec.js [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+    elementCount
+  });
+
+  transform.run();
+
+  const outData = transform.getBuffer('outFloat').getData();
```
Comment (ibgreen) [inline test/modules/core/shadertools/pack.spec.js]: This is not actually a buffer right? Maybe just have a `Transform.getData(`outFloat`)?`

File: modules/imageprocessing/src/modules/pack.js [hunk 4]
```diff
@@ -26,6 +26,7 @@ const DEFAULT_PROPS = {};
 export default {
   name: 'pack',
   fs,
+  vs: fs,
```
[the remark this answers is not in the document]
Comment (ibgreen) [reply modules/imageprocessing/src/modules/pack.js]: https://github.com/uber/luma.gl/blob/e0ad2a0ad0a0653a69e665d1027b45f06da75551/dev-docs/roadmaps/shadertools-roadmap.md#new-shader-type-common

File: test/modules/core/shadertools/pack.spec.js (hunk 3 above)
Comment (1chandu) [reply test/modules/core/shadertools/pack.spec.js]: there is no `getData` API yet, planning on adding along with WebGL1 support as we will be supporting `textures` and `buffers`.

**Pull request merged**